packages/ui/react-ui-editor/src/components/Toolbar/Toolbar.tsx

```diff
@@ -68,19 +72,28 @@ const ToolbarRoot = ({ children, onAction }: ToolbarProps) => {
 type ToolbarButtonProps = {
   Icon: Icon;
   onClick: () => Action | undefined;
+  getState?: (state: Formatting) => boolean;
 } & NonNullable<Pick<ButtonProps, 'title'>>;
 
-const ToolbarButton = ({ Icon, onClick, title }: ToolbarButtonProps) => {
-  const { onAction } = useToolbarContext('ToolbarButton');
+const ToolbarButton = ({ Icon, onClick, title, getState }: ToolbarButtonProps) => {
+  const { onAction, state } = useToolbarContext('ToolbarButton');
   const handleClick = () => {
     const action = onClick();
     if (action) {
       onAction?.(action);
     }
   };
+  const active = getState ? getState(state) : false;
 
+  // TODO: use the correct way to style these as depressed
```

Comment:
Do we have some CSS class I can add to make the buttons look active?

Reply:
i've asked our ux designer to look into this and provide something. in the interim a noisy hack would be to add `ring` which would be good enough for testing/development.

Reply:
I've used a background style for the moment, which makes this look pretty much right, but doesn't seem to be how you do styling in general.

Reply:
right, we use tailwind classes, so would prefer:

`classNames={mx('p-2', active && 'ring')}`